### PR TITLE
Xenowears

### PR DESCRIPTION
**Pull request by FoxxoTrystan**
# Description

Add a bit of clothing specie related and some tiny misc changes.

Normally i wanted to add the spooders changes here but with the other spider specie coming in il hold myself for now.

---

# TODO

- [x] Add clothings
- [x] Loadouts/Vending machines

---

<details><summary><h1>Media</h1></summary>
<p>

image

</p>
</details>

---

# Changelog

:cl:
- add: Xenowears! new non-human related clothings.

### CODE REVIEW

**Comment by DangerRevolution** [comment]
Lots of unrelated changes under one PR; probably worth splitting off some parts.....

File: Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        }
+        query.Dispose();
+    }
+
+    private void UpdateSound(EntityUid uid, EmitsSoundOnMoveComponent component)
+    {
+        if (!_xformQuery.TryGetComponent(uid, out var xform) ||
+            !_physicsQuery.TryGetComponent(uid, out var physics))
+            return;
+
```
Comment (VMSolidus) [inline Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs:62]: xformQuery is deprecated, and the first half of this check is unnecessary now. All systems parented to EntitySystem can instead reference a transform component via 
`Transform(uid)`, which represents the always-available, guaranteed never null, TransformComponent of a given EntityUid.

File: Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        // Space does not transmit sound
+        if (xform.GridUid == null)
+            return;
```
Comment (VMSolidus) [inline Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs:65]:
```suggestion
        if (Transform(uid).GridUid == null)
            return;
```

File: Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+        if (component.RequiresGravity && _gravity.IsWeightless(uid, physics, xform))
+            return;
+
+        var parent = xform.ParentUid;
```
Comment (VMSolidus) [inline Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs:70]:
```suggestion
        var parent = Transform(uid).ParentUid;
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by VMSolidus — CHANGES_REQUESTED** [review]
Also, all 3 test fails are legitimate.

**Comment by DangerRevolution** [comment]
test fail is STILL legitimate

File: Content.Server/Wagging/WaggingSystem.cs [hunk 4]
```diff
@@ -65,6 +65,8 @@ public bool TryToggleWagging(EntityUid uid, WaggingComponent? wagging = null, Hu
 
         wagging.Wagging = !wagging.Wagging;
 
+        _actions.SetToggled(wagging.ActionEntity, wagging.Wagging);
```
Comment (DangerRevolution) [inline Content.Server/Wagging/WaggingSystem.cs:68]: Tail Wagging needs to go into a separate PR

File: Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+using Robust.Shared.GameStates;
+using Robust.Shared.Map;
+
+namespace Content.Shared.Clothing.Components;
+
+/// <summary>
+///   Indicates that the clothing entity emits sound when it moves.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class EmitsSoundOnMoveComponent : Component
```
Comment (DangerRevolution) [inline Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs:11]: This needs to go into a different PR

File: Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+using Content.Shared.Inventory.Events;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Movement.Components;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.Clothing.Systems;
+
+public sealed class EmitsSoundOnMoveSystem : EntitySystem
```
Comment (DangerRevolution) [inline Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs:14]: Refer to above, EmitsSoundOnMove needs to go into a separate PR

File: Resources/Locale/en-US/interaction/interaction-popup-component.ftl [hunk 7]
```diff
@@ -4,6 +4,7 @@
 
 petting-success-generic = You pet {THE($target)} on {POSS-ADJ($target)} head.
 petting-success-soft-floofy = You pet {THE($target)} on {POSS-ADJ($target)} soft floofy head.
+petting-success-generic-others = { CAPITALIZE(THE($user)) } pets {THE($target)}.
```
Comment (DangerRevolution) [inline Resources/Locale/en-US/interaction/interaction-popup-component.ftl:7]: This is weird, petting your co-worker. In #563 consensus is that hugging someone is a little weird; petting is even more touching them randomly?

This can also be emoted.

File: Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml [hunk 8]
```diff
@@ -280,32 +280,32 @@
       state: vulp-fade
 
 - type: marking
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml:282]: As above, tail changes go into a separate PR

File: Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml [hunk 9]
```diff
@@ -377,3 +377,14 @@
     sprite: Clothing/OuterClothing/Misc/red_racoon.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Misc/red_racoon.rsi
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterUnathiRobe
+  name: roughspun robes
+  description: A traditional Unathi garment.
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml:385]: We don't have Unathi on EE (yet?), we have Lizardpeople.

File: Resources/Prototypes/Entities/Clothing/clothwarps.yml [hunk 10]
```diff
@@ -0,0 +1,38 @@
+- type: entity
+  parent: Clothing
+  id: ClothingsClothwarp
+  name: clothwarps
+  description: A roll of treated canvas used for wrapping claws or paws.
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/clothwarps.yml:5]: > Claws or Paws

But this can be worn on regular hands?

File: Resources/Textures/Clothing/Neck/mantles/oldmantle.rsi/meta.json [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/ParadiseSS13/Paradise",
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/Neck/mantles/oldmantle.rsi/meta.json:4]: This needs to reference the commit it was taken from.

File: Resources/Textures/Clothing/Neck/mantles/unathimantle.rsi/meta.json [hunk 12]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/ParadiseSS13/Paradise",
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/Neck/mantles/unathimantle.rsi/meta.json:4]: This needs to reference the commit it was taken from.

File: Resources/Textures/Clothing/OuterClothing/Misc/unathirobe.rsi/meta.json [hunk 13]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/ParadiseSS13/Paradise",
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/OuterClothing/Misc/unathirobe.rsi/meta.json:4]: This needs to reference the commit this was taken from.

File: Resources/Textures/Clothing/Shoes/Misc/clothwarp.rsi/meta.json [hunk 14]
```diff
@@ -0,0 +1 @@
+{"version": 1, "license": "CC-BY-SA-3.0", "copyright": "Taken from https://github.com/ParadiseSS13/Paradise/", "size": {"x": 32, "y": 32}, "states": [{"name": "icon"}, {"name": "equipped-FEET", "directions": 4}, {"name": "equipped-HAND", "directions": 4}]}
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/Shoes/Misc/clothwarp.rsi/meta.json:1]: This needs to reference the commit it was taken from.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml (hunk 9 above)
Comment (FoxxoTrystan) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml:385]: They are the same.

File: Resources/Prototypes/Entities/Clothing/clothwarps.yml (hunk 10 above)
Comment (FoxxoTrystan) [reply Resources/Prototypes/Entities/Clothing/clothwarps.yml:5]: Yeah but generally this is xenowears.

File: Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs (hunk 6 above)
Comment (FoxxoTrystan) [reply Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs:14]: This is needed for this PR, i am not making a new PR just for for this system.

File: Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs (hunk 5 above)
Comment (FoxxoTrystan) [reply Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs:11]: This is needed for this PR, i am not making a new PR just for for this system.

File: Content.Server/Wagging/WaggingSystem.cs (hunk 4 above)
Comment (FoxxoTrystan) [reply Content.Server/Wagging/WaggingSystem.cs:68]: TailWagging changes are extremly mini and in all honest would annoy me to move it to a different PR.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml (hunk 9 above)
Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml:385]: I am aware, change the name of the clothing or change the name of the species.

File: Resources/Prototypes/Entities/Clothing/clothwarps.yml (hunk 10 above)
Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/clothwarps.yml:5]: I don't know what that means.

File: Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs (hunk 5 above)
Comment (DangerRevolution) [reply Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs:11]: Keep this, then move wagging into a different PR.

File: Resources/Locale/en-US/deltav/markings/vulpkanin.ftl [hunk 15]
```diff
@@ -102,14 +102,6 @@ marking-VulpTailTip-vulp = Vulpkanin tail (base)
 marking-VulpTailTip-vulp-tip = Vulpkanin tail (tip)
 marking-VulpTailTip = Vulpkanin (tip)
 
-marking-VulpTailWag-vulp_wag = Vulpkanin tail (base)
```
Comment (Mnemotechnician) [inline Resources/Locale/en-US/deltav/markings/vulpkanin.ftl:105]: This removes a lot of existing markings, but does not provide a db migration. This can lead to unexpected errors at runtime.

Comment (DEATHB4DEFEAT) [reply Resources/Locale/en-US/deltav/markings/vulpkanin.ftl:105]: It shouldn't

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml [hunk 16]
```diff
@@ -67,6 +67,7 @@
     ClothingShoesColorOrange: 2
     ClothingShoesColorRed: 2
     ClothingShoesColorPurple: 2
+    ClothingsClothwarp: 4
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml:70]:
```suggestion
    ClothingClothWrap: 4
```

File: Resources/Prototypes/Entities/Clothing/clothwarps.yml [hunk 17]
```diff
@@ -0,0 +1,38 @@
+- type: entity
+  parent: Clothing
+  id: ClothingsClothwarp
+  name: clothwarps
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Clothing/clothwarps.yml:4]:
```suggestion
  id: ClothingClothWrap
  name: cloth wraps
```

File: Resources/Prototypes/Loadouts/shoes.yml [hunk 18]
```diff
@@ -243,3 +243,11 @@
       species: Harpy
   items:
     - ClothingShoesMiscWhite
+
+- type: loadout
+  id: LoadoutShoesClothwarp
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Loadouts/shoes.yml:248]:
```suggestion
  id: LoadoutShoesClothWrap
```

File: Resources/Prototypes/Loadouts/shoes.yml [hunk 19]
```diff
@@ -243,3 +243,11 @@
...
+  category: Shoes
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingsClothwarp
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Loadouts/shoes.yml:253]:
```suggestion
    - ClothingClothWrap
```

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/clothwarp.yml [hunk 20]
```diff
@@ -0,0 +1,13 @@
+- type: constructionGraph
+  id: ClothingsClothwarp
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/clothwarp.yml:2]:
```suggestion
  id: ClothingClothWrap
```

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/clothwarp.yml [hunk 21]
```diff
@@ -0,0 +1,13 @@
...
+  graph:
+    - node: start
+      edges:
+        - to: warps
+          steps:
+          - material: Cloth
+            amount: 2
+            doAfter: 1
+    - node: warps
+      entity: ClothingsClothwarp
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/clothwarp.yml:13]:
```suggestion
      entity: ClothingClothWrap
```

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/clothwarp.yml [hunk 22]
```diff
@@ -0,0 +1,13 @@
+- type: constructionGraph
+  id: ClothingsClothwarp
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: warps
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/clothwarp.yml:7]:
```suggestion
        - to: wraps
```

File: Resources/Prototypes/Recipes/Construction/clothing.yml [hunk 23]
```diff
@@ -96,3 +96,14 @@
   description: Comfy, yet haunted by the ghosts of ducks you fed bread to as a child.
   icon: { sprite: Clothing/Shoes/Misc/duck-slippers.rsi, state: icon }
   objectType: Item
+
+- type: construction
+  name: clothwarps
+  id: ClothingsClothwarp
+  graph: ClothingsClothwarp
+  startNode: start
+  targetNode: warps
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Recipes/Construction/clothing.yml:105]:
```suggestion
  name: cloth wraps
  id: ClothingClothWrap
  graph: ClothingClothWrap
  startNode: start
  targetNode: wraps
```

File: Resources/Prototypes/Recipes/Construction/clothing.yml [hunk 24]
```diff
@@ -96,3 +96,14 @@
...
+  category: construction-category-clothing
+  description: A roll of treated canvas used for wrapping claws or paws.
+  icon: { sprite: Clothing/Shoes/Misc/clothwarp.rsi, state: icon }
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Recipes/Construction/clothing.yml:108]:
```suggestion
  icon: { sprite: Clothing/Shoes/Misc/clothWrap.rsi, state: icon }
```

File: Resources/Textures/Clothing/Neck/Misc/bellcollar.rsi/meta.json [hunk 25]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by mnemotechnician (GitHub, Frontier) for Frontier Station 14",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Clothing/Neck/Misc/bellcollar.rsi/meta.json:4]:
```suggestion
  "copyright": "Made by mnemotechnician (GitHub)",
```

File: Resources/Textures/Clothing/Shoes/Misc/clothwarp.rsi/meta.json
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Clothing/Shoes/Misc/clothwarp.rsi/meta.json:1]: Un-minify this

**Review by DEATHB4DEFEAT — DISMISSED** [review]
I've only reviewed 1/4 of this.

This should be multiple PRs.

**Comment by VMSolidus** [comment]
Test fails are actually legitimate, but I'm also going to hold off approving this because we're actually about to completely yeet the Clothesmate bloat. 

https://github.com/Simple-Station/Einstein-Engines/pull/637

**Comment by VMSolidus** [comment]
Test fail is legitimate, just change the restock price for the autodrobe and don't worry about it. I will purge the autodrobe's bloat in a separate PR.
image

**Review by VMSolidus — APPROVED** [review]

**Comment by FoxxoTrystan** [comment]
This need to be reviewed and have the "do not merge" removed.

File: Resources/Prototypes/Actions/types.yml [hunk 26]
```diff
@@ -375,4 +375,4 @@
     useDelay: 40
     event:
       !type:FabricateActionEvent
-      fabrication: FoodGumball
+      fabrication: FoodGumball
```
Comment (DangerRevolution) [inline Resources/Prototypes/Actions/types.yml:378]: >:(

File: Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml (hunk 9 above)
Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml:385]: > I am aware, change the name of the clothing or change the name of the species.

File: Resources/Textures/Clothing/Neck/mantles/oldmantle.rsi/meta.json (hunk 11 above)
Comment (DangerRevolution) [reply Resources/Textures/Clothing/Neck/mantles/oldmantle.rsi/meta.json:4]: This is still valid

File: Resources/Textures/Clothing/Neck/mantles/unathimantle.rsi/meta.json (hunk 12 above)
Comment (DangerRevolution) [reply Resources/Textures/Clothing/Neck/mantles/unathimantle.rsi/meta.json:4]: This is still valid

File: Resources/Textures/Clothing/OuterClothing/Misc/unathirobe.rsi/meta.json (hunk 13 above)
Comment (DangerRevolution) [reply Resources/Textures/Clothing/OuterClothing/Misc/unathirobe.rsi/meta.json:4]: This is still valid

File: Resources/Textures/Clothing/Shoes/Misc/clothWrap.rsi/meta.json [hunk 27]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/ParadiseSS13/Paradise/",
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/Shoes/Misc/clothWrap.rsi/meta.json:4]: Commit?

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Textures/Clothing/Shoes/Misc/clothWrap.rsi/meta.json (hunk 27 above)
Comment (FoxxoTrystan) [reply Resources/Textures/Clothing/Shoes/Misc/clothWrap.rsi/meta.json:4]: Holy hell this is annoying, all the others PRs i made nearly has none of it, and only this one you annoy me with.

Gaaah.

**Comment by FoxxoTrystan** [comment]
https://github.com/Simple-Station/Einstein-Engines/blob/794b8ebeaf5db889d008cb0bc1a866f9cc23174b/Resources/Locale/en-US/cargo/bounties.ftl#L114
https://github.com/Simple-Station/Einstein-Engines/blob/794b8ebeaf5db889d008cb0bc1a866f9cc23174b/Resources/Prototypes/Entities/Objects/Devices/translator_implants.yml#L59

**Comment by FoxxoTrystan** [comment]
Sure we can change the specie name, but not in this PR

**Review by DangerRevolution — APPROVED** [review]
i'll rename the gyatt damn species in a different PR
